### ISSUE TITLE
Error when syncing the project

### ISSUE THREAD

**Issue by avshenuk** [bazelbuild/intellij#371]
After upgrading the plugin to version v2018.06.11 there is always an error during the project sync:
```
ERROR: /private/var/tmp/_bazel_serhiiavsheniuk/2b417f85bcff7e9c5c4d4bfc3565535a/external/intellij_aspect/intellij_info_impl.bzl:318:43: name 'cc_common' is not defined (did you mean 'java_common'?).
ERROR: Extension 'intellij_info_impl.bzl' has errors.
```
which preserves in the latest version of plugin "2018.07.09" as well.

Bazel is upgraded to the latest version with `brew`:
```
Build label: 0.15.2-homebrew
Build target: bazel-out/darwin-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Tue Jul 17 13:06:19 2018 (1531832779)
Build timestamp: 1531832779
Build timestamp as int: 1531832779
```

**Comment by avshenuk** [comment]
The problem is with this line:
https://github.com/bazelbuild/intellij/blob/6aae6733c4a770e62775077fe197309e2bd57df0/aspect/intellij_info_impl.bzl#L318

**Comment by avshenuk** [comment]
Does this look like moving forward to the solution of this problem?
https://github.com/bazelbuild/bazel/commit/769f72eb475cea0f4113ac49dd66dc3fda0608de

**Comment by jvoung** [comment]
At first blush, this looks like https://github.com/bazelbuild/intellij/issues/356. Updating bazel worked in that case, but you say that you have a pretty new version 0.15.2? Are you sure that's the one used in the IDE (selected in Settings > Bazel Settings)?

Otherwise, cc_common should have been available earlier than bazelbuild/bazel@769f72e. We don't use the "compile" and "link" API from that commit.

**Comment by avshenuk** [comment]
Sorry, the version used was actually the one tied to the project, not the one in the system. That's why I was confused. Sorry for that.
